src/zenml/integrations/aws/flavors/sagemaker_step_operator_flavor.py

```diff
@@ -29,17 +31,68 @@
 class SagemakerStepOperatorSettings(BaseSettings):
     """Settings for the Sagemaker step operator.
 
-    Attributes:
-        instance_type: The type of the compute instance where jobs will run.
-            Check https://docs.aws.amazon.com/sagemaker/latest/dg/notebooks-available-instance-types.html
-            for a list of available instance types.
+    Non-Estimator attributes:
```

Comment:
The `Attributes` is a specific keyword for the google docstring format that we're using which get parsed into our API docs, so that would need to be reverted.